README.md

```diff
@@ -18,9 +18,11 @@
   <img alt="Kubescape logo" align="right" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/kubescape/stacked/color/kubescape-stacked-color.svg" width="150">
 </picture>
 
-_An open-source Kubernetes security platform for your IDE, CI/CD pipelines, and clusters_
+_An open-source Kubernetes security platform for your IDE, CI/CD pipelines, and clusters that seperates out security signal from scanner noise_
 
-Kubescape is an open-source Kubernetes security platform. It includes risk analysis, security compliance, and misconfiguration scanning. Targeted at the DevSecOps practitioner or platform engineer, it offers an easy-to-use CLI interface, flexible output formats, and automated scanning capabilities. It saves Kubernetes users and admins precious time, effort, and resources.
+Kubescape is an open-source Kubernetes security platform, built for use in your day-to-day workflow, by fitting into your IDE, CI/CD pipelines and clusters. It serves as a one-stop-shop for Kuberenetes security. You can run scans via the CLI, or add the Kubescape Helm chart, which gives an in-depth view of what is going on in the cluster.
```

Comment:
> which gives an in-depth view of what is going on in the cluster
I'd rather rephrase this around monitoring (that the CLI cannot do)

Reply:
Not sure what you meant here. 
The reason I phrased it the way I did is that I wanted to highlight that agentess scans are available and to get more info you install the operator.

Reply:
Changing 
> You can run scans via the CLI, or add the Kubescape Helm chart, which gives an in-depth view of what is going on in the cluster
to something like
"You can run scans via the CLI, or add the Kubescape Helm chart for continuous monitoring of the cluster"